gensim/utils.py

```diff
@@ -358,6 +361,10 @@ def _smart_save(self, fname, separately=None, sep_limit=10 * 1024**2,
         in both Python 2 and 3.
 
         """
+        import os
+        directory = os.path.dirname(fname)
+        if not os.path.exists(directory):
```

Comment:
Definitely not -- no creating of directories please. Non-existent path should end in an exception (as it was before).